salt/utils/gitfs.py

```diff
@@ -1599,9 +1599,15 @@ def _traverse(tree, blobs, prefix):
             _traverse(tree, blobs, self.root(tgt_env))
         add_mountpoint = lambda path: salt.utils.path_join(self.mountpoint(tgt_env), path)
         for repo_path in blobs.get('files', []):
-            files.add(add_mountpoint(relpath(repo_path)))
+            if not salt.utils.is_windows():
+                files.add(add_mountpoint(relpath(repo_path)))
+            else:
+                files.add('/'.join(repo_path.replace('.:\\', '', 1).split(os.sep)))
```

Comment:
Sure, let me grab the powershell script we're using to build hubble on windows. We're using pyinstaller.

I would definitely like another set of eyes. The `.:\` is very mysterious.